Azure/communication-ui-library-ios
# [Feature] Mask admin user from read receipt and participant added message

## Purpose

* Mask admin user from read receipt and participant added message

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test


* Open a chat thread that contains 2 participants plus admin user
* Have the other participant read the last message
* Check the read receipt icon is shown correctly
* On the very top of the chat thread, check there's no join message for admin user


## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |
<img src="https://user-images.githubusercontent.com/107075081/203141937-21caa8f5-b555-49a2-b409-8331d5cb8841.png" width="200" height="400" /> | <img src="https://user-images.githubusercontent.com/107075081/203142033-f6b462d2-7331-413e-b1b9-663b6d0faa49.png" width="200" height="400" />

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |
<img src="https://user-images.githubusercontent.com/107075081/203142240-ec03e997-975c-48e6-a270-2a469bc8a7bf.png" width="200" height="400" /> | <img src="https://user-images.githubusercontent.com/107075081/203142110-3c555da0-9948-4cb7-b8aa-755fcab2b5bc.png" width="200" height="400" />

## Other Information

N/A
